pages/alerts.rst

```diff
@@ -284,11 +284,57 @@ Client URL
 .. image:: /images/alerts_pagerduty_notification.png
 
 
+.. _alert_notification_Slack Notification:
+
+Slack alert notification
+------------------------
+The Slack alert notification allows you to send notifications using your `Webhook URL <https://api.slack.com/messaging/webhooks>`__ to your slack workspace
+in response to the configured event definitions in your Graylog server.
+
+.. image:: /images/alerts_slack_notification.png
+
+These are the supported configuration options:
+
+Webhook URL
+    An unique URL to which you send a JSON payload with the message text and some options.
+
+Channel
+    A `channel <https://api.slack.com/methods/chat.postMessage#channels>`__ to send message to.
+
+Configuration Color
+    Highlight the custom message with this `color <https://api.slack.com/reference/surfaces/formatting#quotes>`__ .
+
+Title
+    A title for this notification.
+
+Description
+    A long description for this slack notification.
+
```

Comment:
These are common to all Notifications and don't need to be included for each one.